HomeWork/HomeWork/RandomUser.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeWork
+{
+    internal class RandomUser
+    {
+        public string CreateRandomLogin()
+        {
+            Random user = new Random();
```

Comment:
maybe we should call the object of Random class, like a "random" or something like this?